source/localizable/components/wrapping-content-in-a-component.md

```diff
@@ -93,6 +93,14 @@ Once yielded the data can be accessed within wrapped content by referencing the
 Finally we want to share the model of the data a user fills out for the post within our `blog-post` and body components.
 To share the `postData` object with the new body component, you can add arguments to the component helper.
 
+```app/components/blog-post.js
```

Comment:
I think binding postData directly into the blog-post component declaration above would be better than giving it internal state.